Description: Fix spelling errors reported by lintian
 Some errors like 'EVERYTIME' and 'some system' do not
 need fixing.
Author: Valentin Vidic <Valentin.Vidic@CARNet.hr>
Last-Update: 2018-04-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/lrm/clientlib.c
+++ b/lib/lrm/clientlib.c
@@ -197,7 +197,7 @@
 	/* if already signed on, sign off first*/
 	if (is_signed_on) {
 		cl_log(LOG_WARNING,
-			"lrm_signon: the client is alreay signed on, re-sign");
+			"lrm_signon: the client is already signed on, re-sign");
 		lrm_signoff(lrm);
 	}
 
--- a/logd/logtest.c
+++ b/logd/logtest.c
@@ -78,7 +78,7 @@
 	snprintf(msgstring,  sizeof(msgstring),"Message %d", count++);
 	fprintf(stderr, "sending %s\n", msgstring);
 	if (LogToDaemon(priority, msgstring,MAXMSGSIZE, FALSE) != HA_OK){			
-		printf("sending out messge %d failed\n", count);
+		printf("sending out message %d failed\n", count);
 		dropmsg++;
 	}
 	
--- a/lrm/lrmd/lrmd.c
+++ b/lrm/lrmd/lrmd.c
@@ -1394,7 +1394,7 @@
 	/*get the client in the client list*/
 	client = lookup_client(pid);
 	if (NULL == client) {
-		lrmd_log(LOG_ERR, "on_connect_cbk: donnot find the client "
+		lrmd_log(LOG_ERR, "on_connect_cbk: do not find the client "
 			"[pid:%d] in internal client list. ", pid);
 		send_ret_msg(ch, HA_FAIL);
 		return TRUE;
@@ -1680,7 +1680,7 @@
 	cl_msg_add_list(ret,F_LRM_RCLASS,ra_class_list);
 	if (HA_OK != msg2ipcchan(ret, client->ch_cmd)) {
 		lrmd_log(LOG_ERR,
-			"on_msg_get_rsc_classes: cannot send the ret mesage");
+			"on_msg_get_rsc_classes: cannot send the ret message");
 	}
 	ha_msg_del(ret);
 
@@ -3363,7 +3363,7 @@
 		} else {
 			lrmd_log(LOG_WARNING, "There is something wrong: the "
 				"first line isn't read in. Maybe heartbeat does "
-				"not ouptut the string correctly for the status "
+				"not output the string correctly for the status "
 				"operation, or perhaps the code is wrong.");
 		}
 	}
--- a/lrm/admin/lrmadmin.c
+++ b/lrm/admin/lrmadmin.c
@@ -129,7 +129,7 @@
         "daemon dead1",
         "daemon dead2",
         "daemon stopped",
-        "status unknow"
+        "status unknown"
 };
 
 
--- a/lib/clplumbing/cl_msg_types.c
+++ b/lib/clplumbing/cl_msg_types.c
@@ -1015,7 +1015,7 @@
 		
 	} else { 
 		cl_log(LOG_ERR, "field already exists "
-		       "with differnt type=%d", msg->types[j]);
+		       "with different type=%d", msg->types[j]);
 		return (HA_FAIL);
 	}
 		
